### ISSUE TITLE
[Bug]: Navidrome is splitting an album into multiple albums

### ISSUE THREAD

**Issue by mneuburg-livefront** [navidrome/navidrome#2867]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

0.51.0 (fd61b29a)

### Current Behavior

I have an album which Navidrome persists in categorizing into two albums with identical names (Boccherini Cello Concertos).

### Expected Behavior

I expect all songs tagged with the album name Boccherini Cello Concertos to be categorized by Navidrome as a single album.

### Steps To Reproduce

Run Navidrome against a folder containing the two enclosed songs. Navidrome will create two separate albums named called Boccherini Cello Concertos. I can see no justification for this behavior; the tags are completely consistent between the two songs.

### Environment

```markdown
- OS: Mac OS 10.13.6 (17G14042)
- Browser: Safari
- Client: iSub, but it doesn't matter, I've tried several
```

### How Navidrome is installed?

Binary (from downloads page)

### Configuration

```toml
MusicFolder = '/Volumes/TomThumb/music'
ScanSchedule = '@every 12h'
EnableExternalServices = false
DataFolder = '/Volumes/Humlet/Users/matt/navidrome'
Jukebox.Enabled = true
MPVPath = '/Applications/mpv.app/Contents/MacOS/mpv'
```

### Anything else?

[tracks.zip](https://github.com/navidrome/navidrome/files/14323091/tracks.zip)

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by deluan** [comment]
Have you checked out https://www.navidrome.org/docs/faq/#-i-have-an-album-with-tracks-by-different-artists-why-is-it-broken-up-into-lots-of-separate-albums-each-with-their-own-artist ?

TL;DR: The tags that can influence the split of an album are `albumartist`, `compilation`, and dates and year.

Also take a look at this similar (if not the same) issue: https://github.com/navidrome/navidrome/issues/2728

**Comment by mneuburg-livefront** [comment]
@deluan I did see that "similar" issue, and that is why I have submitted this one; the "similar" issue was claimed as fixed, but it is not fixed in my case. — The tracks in question have no album artist, they are both compilation, and they have no date tags. I have tried regenerating the tracks and regenerating the entire navidrome database but I get the same result. That is why I include the tracks in question; maybe you can see a difference between them that I cannot see.

**Comment by deluan** [comment]
Please send me two tracks, one from each of the albums. Send it to deluan @ navidrome . org, and I'll take a look at what could be the issue here.

**Comment by mneuburg-livefront** [comment]
Yes, here are screen shots.

![Screenshot 2024-02-18 at 10 10 31 AM](https://github.com/navidrome/navidrome/assets/125824749/39586087-d25c-4b37-998a-2eaae419ed51)

![Screenshot 2024-02-18 at 10 10 40 AM](https://github.com/navidrome/navidrome/assets/125824749/02087d9d-4e89-473e-b48b-0f539c7bf67e)

![Screenshot 2024-02-18 at 10 11 16 AM](https://github.com/navidrome/navidrome/assets/125824749/09be3067-43d4-4106-9b81-9ed92ff6aaca)

The blue color of the artist entries is kind of weird, plus actually (this seems to be new) they are appearing in two _different_ albums (with the same name).

**Comment by mneuburg-livefront** [comment]
Hang on a tic - I have a suspicion that I've got two copies of these files!

**Comment by mneuburg-livefront** [comment]
Yep, that was it! I had two copies, in two different folders. So we can scratch this one, although I wonder whether Navidrome might be able to do more to help track down this kind of thing (though I leave that to your imagination, as to how / whether it might do that).

**Comment by deluan** [comment]
> although I wonder whether Navidrome might be able to do more to help track down this kind of thing

I think this is hard to accomplish, although you should have seen two copies of the songs in Navidrome.

Anyway, closing this as it is a non-issue.
